### PR TITLE
958 new start page

### PR DESCRIPTION
**Pull request by jarekzet**
New home page layout based on projects from #958

### CODE REVIEW

**Comment by jarekzet** [comment]
Pushed content modifications.

**Comment by bwilk** [comment]
@goreck888 fallback for empty opinions and other potentially null items is needed

File: app/views/home/index.html.haml [hunk 1]
```diff
@@ -106,42 +107,24 @@
...
-                  %i.fas.fa-star{ value: 4 }
-                %li.list-inline-item
-                  %i.far.fa-star{ value: 5 }
-                %li.list-inline-item
-                  %span.ml-1.font-weight-bold 4.75/5
-          .card-description
-            It's a good looking application made with decent quality for the price, but if you need something longer
-            lasting, you should look elsewhere.
-
+      = render "opinion", opinion: @opinion || "There are no reviews available."
```
Comment (bwilk) [inline app/views/home/index.html.haml]: I see error `undefined method `project_item' for "There are no reviews available.":String`

Comment (goreck888) [reply app/views/home/index.html.haml]: Done.

**Comment by roksanaer** [comment]
I'm also reviewing the ENglish side of the start page (along with ROB from trust IT). Here:
https://docs.google.com/document/d/1f0QMS-fo7Lb2-3-pqkQ9VORVk4p2pr9OYn8b7eJ9yAs/edit
is the google doc where revised version will be available (already found some obvious spelling mistakes...) Please wait for the comment here until it's ready, then include the changes.

**Pull request merged**